presto-accumulo/pom.xml

```diff
@@ -0,0 +1,357 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>0.152-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-accumulo</artifactId>
+    <description>Presto - Accumulo Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <accumulo.version>1.6.2</accumulo.version>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <curator.version>2.9.1</curator.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>javax.ws.rs</groupId>
+                <artifactId>javax.ws.rs-api</artifactId>
+                <version>2.0.1</version>
```

Comment:
This isn't necessary. It's already defined in airbase (the parent POM of the entire Presto project).